webpack/assets/javascripts/react_app/components/common/forms/NumericInput.js

```diff
@@ -11,16 +12,25 @@ const NumericInput = ({
   value,
   onChange,
   format,
+  parser,
+  step,
   precision,
   minValue,
+  disabled,
+  readOnly,
 }) => (
-  <CommonForm label={label} className={`common-numericInput ${className}`}>
-    <ReactNumericInput
-      format={format}
+  <CommonForm label={label} className={`foreman-numeric-input ${className}`}>
```

Comment:
I could find two dives with the `foreman-numeric-input` class-name on the dom.
I guess only one of them should receive the css properties defined in the scss file?